docs/install.md

```diff
@@ -89,37 +93,58 @@ To install Nextflow:
     :::
 
     :::{warning}
-    Nextflow will update its executable during the self update process, therefore the update can fail if the executable is placed in a directory with restricted permissions.
+    Nextflow updates its executable during the self-install process, therefore the update can fail if the executable is placed in a directory with restricted permissions.
     :::
 
-4. Confirm that Nextflow is installed correctly:
+4. Confirm Nextflow is installed correctly:
 
     ```{code-block} bash
     :class: copyable
     nextflow info
     ```
 
-## Seqera Platform
+### Conda
 
-You can launch workflows directly from [Seqera Platform](https://seqera.io/platform/) without installing Nextflow locally.
+To install Nextflow with Conda:
 
-Launching from Seqera Platform provides you with:
+1. Add the `bioconda` and `conda-forge` channels to your Conda configuration:
+    
+    ```{code-block} bash
+    :class: copyable
+    conda config --add channels bioconda
+    conda config --add channels conda-forge
+    ```
```

Comment:
This is not strictly required. It should be enough specifying `bioconda::nextflow` when installing it

Reply:
I've removed this step and adjusted the other steps to account for specifying `bioconda::nextflow`. Thanks!

Reply:
ok, thanks